### DISCUSSION TITLE
Update to 1.21

### DISCUSSION THREAD

**Discussion by BHilde1119** [andersmmg/LockAndBlock#16] [Ideas]

**Comment by andersmmg** [comment]
My mods mainly just support 1.20.1 for now as supporting multiple versions at once is more than I have time for. They will eventually be updated to a later version when there is another popular modding version, but no guarantees or ETA

**Comment by andersmmg** [comment]
0.10.0 was just released for 1.21.1, I'll consider later versions if they get popular enough for modding
